package.json

```diff
@@ -40,6 +40,7 @@
   "devDependencies": {
     "browserify": "^13.0.0",
     "d3": "3.5",
+    "lodash.result": "^4.4.0",
```

Comment:
That said, I just checked it out and lodash.result is small and well-encapsulated. Might make sense to just use it. Either way (a) I like this and (b) I think it would make sense to take it further and do some diagnostics to warn about issues with type coercion and potential problems with the "natural order" requirement regardless of the accessor being used.